bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "isMobile",
-  "version": "0.3.5",
+  "version": "0.3.6",
```

Comment:
`version` is no longer supported by bower.

https://github.com/bower/bower.json-spec#version